<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>query</class>
 <widget class="QWidget" name="query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>信息查询</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>761</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="functionhorizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QPushButton" name="queryclient_pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>顾客信息</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="querymedicine_pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>药品信息</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="queryorder_pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>订单信息</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="querypurchase_pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>采购信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>100</y>
     <width>781</width>
     <height>151</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="clientpage">
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>611</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="clientgridLayout">
      <item row="0" column="4">
       <widget class="QComboBox" name="csex_comboBox">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>男</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>女</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="caddress_lineEdit">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="caddress_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>住址：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="cname_lineEdit">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="cname_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>顾客姓名：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="client_query_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="cphone_lineEdit">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="cphone_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>电话:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="csex_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>性别：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="clear_client_pushButton">
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="medicinepage">
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>761</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <layout class="QGridLayout" name="medicinegridLayout">
      <item row="0" column="4">
       <widget class="QLabel" name="mmode_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>服用方法：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mname_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>药品名称：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="mdosageForm_lineEdit">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mname_lineEdit">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="mdosageForm_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>剂型：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="mspecification_lineEdit">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="mtype_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>类型： </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="mmode_comboBox">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>外用</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>内服</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="mtype_comboBox">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>中成药</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>西药</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OTC</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="mspecification_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>规格：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="menterprise_lineEdit">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="menterprise_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>    生产企业:    </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="medicine_query_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="clear_medicine_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="orderpage">
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>581</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <layout class="QGridLayout" name="ordergridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="cname_lineEdit_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDateTimeEdit" name="dead_dateTimeEdit">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="dead_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>终止日：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="agency_comboBox">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="order_query_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="cname_label_2">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>顾客姓名：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="start_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>起始日：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="agency_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>经办人：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="start_dateTimeEdit">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="clear_order_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="purchasepage">
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>581</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <layout class="QGridLayout" name="ordergridLayout_2">
      <item row="2" column="3">
       <widget class="QPushButton" name="clear_purchase_pushButton">
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QDateTimeEdit" name="dead_dateTimeEdit_1">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="agency_label_1">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>经办人：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="start_dateTimeEdit_1">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="purchase_query_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="dead_label_1">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>终止日：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="agency_comboBox_2"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_6">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="start_label_1">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>起始日：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>761</width>
     <height>291</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="export_pushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>导出到Excel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
